client/src/pages/insights/Show.js

```diff
@@ -140,7 +140,7 @@ const InsightsShow = ({ pageDispatchers, searchQuery, setSearchQuery }) => {
   const orgQuery = currentUser.isAdmin()
     ? {}
     : {
-      organizationUuid: currentUser.position.organization.uuid,
+      organizationUuid: currentUser?.position?.organization?.uuid,
```

Comment:
This is only used by the Pending Assessments insight, which will now by default give the pending assessments for *all* organizations if the current user doesn't have a position. That's unwanted. We should probably filter out that option from the list of insights for positionless users (they don't need that insight anyway, since they don't have any counterparts themselves until they get assigned a position).